Mathematics/Calculus_and_Analysis/Real_Analysis 2.tex

```diff
@@ -117,82 +117,91 @@ \section{Topology in $\R$}
 \end{enumerate}
 \end{theorem}
 \begin{proof}
-    (1) Let $I_\alpha$ be open, $\any \alpha \in A$, $I = \bigcup\limits_{\alpha \in A} I_\alpha$, $x \in I \Rightarrow x \in I_\alpha$, for some $\alpha \in A$. \\
-    Thus, $\exist \epsilon >0$ s.t $V_\epsilon (x) \subseteq I_\alpha \subseteq I$. $\hfill\blacksquare$ \\
-    (2) Let $I_\alpha$ be open, $\alpha \in \{1,2,...,n\}$, let $\epsilon = min\{ \epsilon_1,...,\epsilon_n\}$, we have $V_\epsilon (x) \subseteq V_{\epsilon i} (x)\subseteq I_\alpha$, $\any i \Rightarrow V_\epsilon (X) \subseteq I$.
+    (1) Let $I_\alpha$ be open, 
+    \[
+    \any \alpha \in \mathcal{A}, I = \bigcup\limits_{\alpha \in A} I_\alpha, x \in I \Rightarrow x \in I_\alpha, \text{for some } \alpha \in \mathcal{A}.
+    \] \\
+    Thus, $\exist \epsilon >0$ s.t $\mathcal{V}_\epsilon (x) \subseteq I_\alpha \subseteq I$. $\hfill\blacksquare$ \\
+    (2) Let $I_\alpha$ be open, $\alpha \in \{1,2,...,n\}$, let $\epsilon = min\{ \epsilon_1,...,\epsilon_n\}$, we have 
+    \[
+    \mathcal{V}_\epsilon (x) \subseteq \mathcal{V}_{\epsilon i} (x)\subseteq I_\alpha, \any i \Rightarrow \mathcal{V}_\epsilon (x) \subseteq I.
+    \]
 \end{proof}
 
 \begin{definition}
-    x is a \uline{limit point of A} if $\any \epsilon > 0$, $V_\epsilon(\epsilon \cap A) \setminus \{x\} \neq \emptyset$, $a_n \neq x$.
+    x is a \uline{limit point of $\mathcal{A}$} if $\any \epsilon > 0$, $V_\epsilon(\epsilon \cap \mathcal{A}) \setminus \{x\} \neq \emptyset$, $a_n \neq x$.
 \end{definition}
 \begin{theorem}
-    x is a LP of $\mathcal{A}$ $\Leftrightarrow \exist \{ a_n \}_{n=1}^\infty \subseteq \mathcal{A}$ s.t $a_n \neq x$, $\any n \in \N$, and $\lim_{n\to\infty} a_n = x$.
+    x is a L.P. of $\mathcal{A}$ $\Leftrightarrow \exist \{ a_n \}_{n=1}^\infty \subseteq \mathcal{A}$ s.t $a_n \neq x$, $\any n \in \N$, and $\lim_{n\to\infty} a_n = x$.
 \end{theorem}
 \begin{proof}
     ($\Rightarrow$) \\
-    Let $\epsilon_n = \frac{1}{n}$, and form a sequence {$a_n$} by picking $a_n \in V_{\epsilon}(x)\cap\mathcal{A}$. \\
+    Let $\epsilon_n = \cfrac{1}{n}$, and form a sequence {$a_n$} by picking $a_n \in V_{\epsilon}(x)\cap\mathcal{A}$. \\
```

Comment:
Haha if I confuse you that you should use `\cfrac` for any fraction, sorry for it